languageserver/test/index.test.ts

```diff
@@ -308,6 +308,20 @@ describe("diagnostics", () => {
       )
   )
 
+  test("InternalEVM contract exists", async() =>
+      testCode(
+          `
+            access(all)
+            fun main() {
+                // Checks that the InternalEVM contract exists
+                // Also that it has the correct value (i.e. the run function exists)
+                log(InternalEVM.run)
```

Comment:
The current implementation defines a `StandardLibraryValue` in [`FVMStandardLibraryValues`](https://github.com/onflow/cadence-tools/pull/332/files#diff-e4908bc6e5534823f44d290f658cbbbb8503ce1126ea58fd611503bbfcb3865dR29) and then declares it in general, for all locations in [`WithExtendedStandardLibraryValues`](https://github.com/onflow/cadence-tools/pull/332/files#diff-9dc030944615b63c919527ae35f799e2c63ff5137f18ef2165d07d808fd2b81fR295-R296).

There are two problems with this:
1. `InternalEVM` is just a current implementation detail of `EVM`. "normal" users of Cadence should not have to know about it, nor use it – we do not guarantee that this implementation detail will stay like it is, we might change the implementation of the `EVM` contract.
2. In addition to defining the `EVM` value, also the `EVM` type needs to be defined.

Both problems can be solved by using FVM's `stdlib.SetupEnvironment` function.

Reply:
Ah gotcha, I didn't realize that `InternalEVM` was only exposed to the EVM contract.  I'll figure out how to update this 🙂

Reply:
I had another quick look into this, but the LS is not using the `runtime` package (`Runtime`/`Environment`), so using `stdlib.SetupEnvironment` isn't really possible at the moment.

We should eventually refactor the LS, but for now, defining `InternalEVM` everywhere to allow the `EVM` contract to be imported seems good enough.